### DISCUSSION TITLE
Sending multiple SysEx arrays with one sendSysEx

### DISCUSSION THREAD

**Discussion by RiNoize** [FortySevenEffects/arduino_midi_library#275] [Q&A]
case 33: // Step Seq Pattern----------------  1  ----------Kick drum 1,5,9,13-----------------------//
        SSpatterns[11]={0x1b}; SSpatterns[16]={0x01}; // 1
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x1e}; SSpatterns[16]={0x00}; // 2
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x21}; SSpatterns[16]={0x00}; // 3
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x24}; SSpatterns[16]={0x00}; // 4
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x27}; SSpatterns[16]={0x01}; // 5
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x2a}; SSpatterns[16]={0x00}; // 6
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x2d}; SSpatterns[16]={0x00}; // 7
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x30}; SSpatterns[16]={0x00}; // 8
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x33}; SSpatterns[16]={0x01}; // 9
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x36}; SSpatterns[16]={0x00}; // 10
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x39}; SSpatterns[16]={0x00}; // 11
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x3c}; SSpatterns[16]={0x00}; // 12
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x3f}; SSpatterns[16]={0x01}; // 13
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x42}; SSpatterns[16]={0x00}; // 14
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x45}; SSpatterns[16]={0x00}; // 15
        MIDI.sendSysEx(18, SSpatterns, true);
        SSpatterns[11]={0x48}; SSpatterns[16]={0x00}; // 16
        MIDI.sendSysEx(18, SSpatterns, true);
        break;            

Hello, as you can see, I am triggering 16 SysEx arrays with a key (case 33:) of a qwerty keyboard using the PS2 key advanced library, is there another way to write this sentence? using only one SendSysEx.
I use Arduino AVR going out by serial TX.
Thanks

**Comment by franky47** [comment]
Short of using a for loop and moving your injected bytes into an array, which would make the core cleaner to read but send the same amount of SysEx, I don't see how you'd do it any differently.
